build.sh

```diff
@@ -207,33 +207,33 @@ EOF
     cmake -B ${build_dir} -G "${generator}" -S "${SCRIPT_DIR}"
     if [[ -z ${skip_build_cmake} ]]; then
       if [[ ! -z "${ninja}" ]]; then
-          ninja -C ${build_dir}
+          ninja -C ${build_dir} -j "$(nproc)"
```

Comment:
Not actually necessary to pass along -j to ninja, since we only want the same number of jobs as cpu count, anyways (default if -j is unspecified is #cpu)

Reply:
oh, TIL!